.github/workflows/deploy-secrets.yml

```diff
@@ -39,6 +39,7 @@ jobs:
       ESDC_ENVIRONMENT_CODE: ${{ secrets.ESDC_ENVIRONMENT_CODE }}
       SFAS_RECEIVE_FOLDER: ${{ secrets.SFAS_RECEIVE_FOLDER }}
       INSTITUTION_REQUEST_FOLDER: ${{ secrets.INSTITUTION_REQUEST_FOLDER }}
+      INSTITUTION_RESPONSE_FOLDER: ${{ secrets.INSTITUTION_RESPONSE_FOLDER }}
```

Comment:
Not PR related but I believe that we can have a ticket for it. 
When we moved the env variables to git there was no option to create regular variables (not protected).
This feature was in beta till a few weeks ago and now it seems that it is available.
I would suggest that we have a ticket to move the non-sensitive env variables to regular env variables.
@andrepestana-aot @guru-aot @ann-aot @sh16011993  Thoughts? 
![image](https://github.com/bcgov/SIMS/assets/61259237/1acdb9a1-0d7b-40cc-b4d8-d5e9c4a654a2)

Reply:
I created the env secret `INSTITUTION_RESPONSE_FOLDER` in GitHub with the value `Institution-Response`. Is it the expected value, right?

Reply:
Yes. The value is correct.

But if we need to use these variables from github actions, I assume we cannot use `{{secrets.something}}` it it should be {{vars.something}} right?

Reply:
Yes, we need to review all the ones that are not sensitive and update the workflows accordingly.

Reply:
Would it be a problem to do this change later? There is a discussion I need to have with David Malcolm regarding best practises regarding secrets/environment variables and I don't want to hold up this PR.

Reply:
@JasonCTang the PR was already merged. There is a change for only one env variable as a proof of concept.
The idea is not to convert all env variables to plain text, only the non-sensitive ones.
The only goal would be to make it easier to identify their values. If the Ministry does not see any benefit on it we can keep them as they are right now.